src/request_rate_manager.cc

```diff
@@ -240,7 +240,8 @@ RequestRateManager::PauseWorkers()
 }
 
 void
-RequestRateManager::ConfigureThreads(const size_t request_count)
+RequestRateManager::ConfigureThreads(
+    const size_t request_count, size_t dataset_offset)
```

Comment:
Still related to [this](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2058826068).

CustomRequestScheduleManager needs to pass the dataset offset down through RequestRateManager through RequestRateWorker to InferContext where it's used to start the benchmark after the payloads that were already sent during warmup.